### PR TITLE
Added a loader and disabled autoscale button in paused state.

### PR DESCRIPTION
**Pull request by Ritika8081**
I've implemented a loader and disabled the autoscale button while the stream is in a paused state

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -82,3 +82,34 @@
...
+  position: fixed;
+  top: 70%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 9999; /* Ensure the loader is on top of other content */
+}
+
+.hidden {
+  display: none;
+}
```
Comment (akadeepesh) [inline src/app/globals.css]: Built in class

File: src/components/Connection.tsx [hunk 2]
```diff
@@ -166,19 +166,25 @@ const Connection: React.FC<ConnectionProps> = ({
...
             <p>Device: {formatPortInfo(port.getInfo())}</p>
             <p>Baud Rate: 57600</p>
           </div>
         ),
       });
+
+      // Show confirmation box
+      window.alert(
+        "Remaining channels will reflect data from channels connected to sensors."
+      );
```
Comment (akadeepesh) [inline src/components/Connection.tsx]: Use shadcn alert dialog, or alert

File: src/components/LandingComp/FAQSection.tsx [hunk 3]
```diff
@@ -103,8 +103,8 @@ const FAQSection = () => {
       question: "What data format does Chords support?",
       answer: (
         <>
-          Chords supports an array format: [counter, A0, A1, ..., A6], where
-          counter is a uint8_t (0-255) and A0-A6 are raw signal values. Array
+          Chords supports an array format: [counter, A0, A1, ..., A5], where
+          counter is a uint8_t (0-255) and A0-A5 are raw signal values. Array
```
Comment (akadeepesh) [inline src/components/LandingComp/FAQSection.tsx]: A4

**Review by akadeepesh — CHANGES_REQUESTED** [review]

**Comment by Ritika8081** [comment]
I have updated code

**Review by akadeepesh — APPROVED** [review]

**Pull request merged**